packages/leafygreen-provider/src/LeafyGreenProvider.tsx

```diff
@@ -1,13 +1,24 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import UsingKeyboardProvider from './UsingKeyboardProvider';
+import TypographyScaleProvider from './TypographyScaleProvider';
 
 interface LeafyGreenProviderProps {
   children: React.ReactNode;
+  typescale?: number;
```

Comment:
Hmm... I'm thinking this may not be descriptive of what this actually is.

`baseFontSize` or `bodyFontSize` might be more accurate.

Reply:
I like `baseFontSize` that's fine

Reply:
should I change the hook/provider names as well or do you think those names still make sense?

Reply:
We should be consistent and update those too.

Reply:
👍